circle.yml

```diff
@@ -1,12 +1,9 @@
 machine:
   node:
     version: 4.2.1
-  ruby:
-    version: 2.2.3
 
 dependencies:
   pre:
-    - gem install github_changelog_generator
```

Comment:
+1 for improved speed. Once this app is Dockerized we'll be able to cache dependencies for breezy and eliminate the external build dependency.